tests/Filesystem/FileTest.php

```diff
@@ -396,6 +405,41 @@ public function testMime()
 		$this->assertSame('text/plain', $file->mime());
 	}
 
+	/**
+	 * @covers ::model
+	 * @covers ::setModel
+	 */
+	public function testModel()
+	{
+		$parent = \Kirby\Cms\Page::factory(['slug' => 'test']);
+		$model = \Kirby\Cms\File::factory([
```

Comment:
Could you move these imports for `Page` and `File` to `use` imports at the top of the file?

```php
use Kirby\Cms\File as CmsFile;
use Kirby\Cms\Page;
```

Same for the `testInvalidModel()` method.